frontend/src/components/config/App.tsx

```diff
@@ -3,7 +3,7 @@ import { Switch } from 'react-router-dom';
 import MainLayout from '../layout/Main';
 import LandingPage from '../../views/Landing';
 import NotFound from '../../views/NotFound';
-import { CustomRoute } from './Route';
```

Comment:
Thanks for taking care of these ESLint warnings :)

Reply:
No problem 🙂